packages/aws-cdk/lib/init-templates/app/java/src/main/java/com/myorg/%name.PascalCased%App.template.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Arrays;
 
 public class %name.PascalCased%App {
-    public static void main(String[] args) {
+    public static void main(final String[] args) {
```

Comment:
`static`s in Java cannot je overloaded so one can consider than `static` implies `final`. From this perspective, I would not explicitly mark it (I have known linters that flag this as a problem)